packages/integration-test/test/node-client.test.ts

```diff
@@ -273,7 +273,7 @@ describe('Request client using a request node', () => {
     await waitForConfirmation(requestDataCancel);
 
     // get requests without boundaries
-    let requests = await requestNetwork.fromTopic(topicsRequest1and2[0]);
+    let requests: Request[] = (await requestNetwork.fromTopic(topicsRequest1and2[0])) as Request[];
```

Comment:
I wonder if we should check the type with an `if` or even assert the type with an `expect` instead of casting. Especially if you see this test as example usage code. That said, I recognize we're trying to ship this fast - I'm open to dissent.

Reply:
Or like you did in packages/request-client.js/test/index.test.ts

```ts
      const response = await requestNetwork.fromTopic('my amazing test topic');
      const requests = Array.isArray(response) ? response : response.requests;
      expect(requests).not.toHaveLength(0);
```

Reply:
Sure, I will fix it